README.md

```diff
@@ -31,6 +31,7 @@ Syntax  | Usage
 ```corporate needs you to find the difference between [register/value] and [register/value]``` | Compares both arguments and jumps to "they're the same picture" if both values are equal
 ```no, I don't think I will``` | Exits the program
 ```perfectly balanced as all things should be``` | Restore balance to your code by randomly deleting half the commands at compile-time
+```what can I say except [character/8-bit register/value]``` | Prints the given character to stdout
```

Comment:
I would change "value" to "ASCII value" to make it more clear that ASCII-codes are supported